mdapi/database/main.py

```diff
@@ -42,18 +43,25 @@
 from mdapi.database.base import compare_databases, index_database
 
 
-def list_branches(status="Active"):
-    urlx = f"{standard.PKGDB2_URL}/api/collections?clt_status={status}"
-    resp = requests.get(urlx, verify=standard.PKGDB2_VERIFY)  # noqa : S113
+def list_branches(status="current"):
+    """
+    As of 03rd September 2024,
+    list_branches("current") returns ['epel10', 'epel8', 'epel9', 'epel9-next', 'f39', 'f40']
+    list_branches("pending") returns ['eln', 'f41', 'f42', 'rawhide']
+    list_branches("frozen")  returns ['f41']
+    """
+    urlx = f"{standard.BODHI_URL}/releases"
+    resp = requests.get(urlx, params={"state": status})  # noqa : S113
     resp.raise_for_status()
-    data = resp.json()
+    data = list(set([item["branch"] for item in resp.json()["releases"]]))
```

Comment:
Instead of converting to a set to remove duplicates, we can filter the desired branches by specifying valid values for id_prefix.

```suggestion
    data = [
        item["branch"]
        for item in resp.json()["releases"]
        if item["id_prefix"] in ("FEDORA", "FEDORA-EPEL", "FEDORA-EPEL-NEXT")
    ]
```

This also would remove the duplication of f42 and rawhide.